cli/cmd/inject.go

```diff
@@ -254,6 +256,7 @@ func InjectYAML(in io.Reader, out io.Writer, version string) error {
 				DNSNameOverride = LocalhostDNSNameOverride
 			}
 			obj = &deployment
+			owner = deployment.Name
```

Comment:
I believe you can just use `meta.Name` instead of repeating this.

Reply:
unfortunately, TypeMeta only contains Kind and APIVersion.